src/shared/components/button/button.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+type ButtonProps = {
```

Comment:
Lets try to implement button that has all default button props with rest operator (onClick, type it is default prop).  We can alsso pass a ref. In the future it could be related to input for example

Reply:
fixed